### ISSUE TITLE
Missing key when adding a new menu entry

### ISSUE THREAD

**Issue by IanTrudel** [freeplane/freeplane#133]
I added `<Entry name="Alignments" />` to [mindmapmodemenu.xml](https://github.com/freeplane/freeplane/blob/1.7.x/freeplane/src/external/resources/xml/mindmapmodemenu.xml). The error reporter is displaying the following:

```
at 2019-04-23 17:57:30 CMT,  hash 04b6d28819c2de2bc7ff3ebe50d0beeb
previous report : no reports sent before
freeplane_version = 1.7.7; freeplane_xml_version = freeplane 1.7.0
git revision = 2c57d361d745d3a56b31d76777b58eac3d4a119b Dimitry Polivaev 2019-03-10T10:37:29Z
java_version = 1.8.0_201; os_name = Linux; os_version = 3.10.0-957.10.1.el7.x86_64
missing key Alignments
missing key Alignments
```

How could it be fixed?

ref. https://sourceforge.net/p/freeplane/discussion/758437/thread/38483188/

**Comment by IanTrudel** [comment]
Copying [Resources_en.properties](https://github.com/freeplane/freeplane/blob/aff7056f4d853dffcdc191d5df4bf483ad86b216/freeplane/src/viewer/resources/translations/Resources_en.properties) in `~/.config/freeplane/1.7.x/resources/translations` and adding the entry to the file fixed the error.

**Comment by vboerchers** [comment]
Do you really consider this a bug?

**Comment by IanTrudel** [comment]
Yes, absolutely. The reasons are simple: it causes a popup window to show up every time Freeplane is started, the fix required knowledge of Java and Resource_en.property is only available on Github. It's just not Groovy scripting friendly.

**Comment by vboerchers** [comment]
I'm not sure what the best way to fix it would be since we really don't want features be added without considering translations. Would a better popup message which assists in fixing the issue be okay?

**Comment by IanTrudel** [comment]
The best way would be to automatically add the missing key to the dictionary when processing `mindmapmodemenu.xml` on the Java side. It will allow Groovy scripters to add new menus without having to worry about popups, failure and/or having to dig up a solution.

**Comment by vboerchers** [comment]
@dpolivaev what do you think about Ian's proposal?

**Comment by vboerchers** [comment]
@BackOrder I have talked with Dimitry. We are not sure why one should add something to the mindmapmodemenu.xml. For scripts and especially for add-ons you have other possibilities to contribute to the menu. There is documentation for mindmapmodemenu.xml since people showed interest in rearranging the menus but they had no need to add something to the menu.

So please elaborate on your use-case.
